### DISCUSSION TITLE
No authentication with Tumblr

### DISCUSSION THREAD

**Discussion by TomInVa57** [TumblThreeApp/TumblThree#451] [Q&A (Question & Answer)]
This started last Sat, 7/8/23 after Tumblr was up and down for a long time during the day, some blogs said it was an update or something that went wrong. Since then, I am not able to authenticate thru TumblThree. Clicking on "authenticate" in the "Connection" tab, it opens the window and allows me to login, I wait a few moments before closing the "authenticate" window, but it never shows logged in. I've logged out of Tumblr.com several times and logged back in. I've logged out thru this app and all web browsers, cleared caches and cookies etc. I can log into the website with no issues or errors. Ideas? Did Tumblr change something (again lol). Thanks in advance.

**Comment by Someonemustnothavethis** [comment]
I'm on Tumblr daily (I know, I know....I'm hooked) and have experienced no problems.  I also post hundreds of pictures most days with no problems.

**Comment by johnflint1985** [comment]
> This started last Sat, 7/8/23 after Tumblr was up and down for a long time during the day, some blogs said it was an update or something that went wrong. Since then, I am not able to authenticate thru TumblThree. Clicking on "authenticate" in the "Connection" tab, it opens the window and allows me to login, I wait a few moments before closing the "authenticate" window, but it never shows logged in. I've logged out of Tumblr.com several times and logged back in. I've logged out thru this app and all web browsers, cleared caches and cookies etc. I can log into the website with no issues or errors. Ideas? Did Tumblr change something (again lol). Thanks in advance.

Same here

**Comment by GregoryWest501** [comment]
I am having the same issue.  I am using Chrome Version 120.0.6099.71 is that makes a difference.    What browser/Version are others using?

**Comment by QldAu** [comment]
Same Result.
I have found that if you Authenticate the new window is spawned and you are in your profile.
If you leave the it for a few mins, the application, greys out the authentication button, and supplys the loggout.
however as soon as you close the authentication window, it returns to the pre authenticated state.
V 2.13.0.442

**Reply by GregoryWest501** [reply]
let me see if I understand you.    I need to authenticate, but instead of closing the tumblr.com window, just wait for tumbltree to do it?

I'll try that today.   love the program, miss it not working...

**Comment by TomInVa57** [comment]
Well. almost a year since I first posted this and still having the same issue. It will authenticate when I first open the program, but then looses it after a few minutes. None of the above has worked. As a last ditch try, I replaced my SDD with a new one, installed Win 10 Pro, installed TumblThree (nothing else), still same issue. :(
